modules/cluster-logging-configuring-image-about.adoc

```diff
@@ -30,7 +30,7 @@ Promtail is not officially supported at this time.
 ====
 
 ////
-RSYSLOG_IMAGE=registry.redhat.io/openshift4/ose-logging-rsyslog:v4.3 <6>
+RSYSLOG_IMAGE=registry.redhat.io/openshift-logging/rsyslog:v4.3 <6>
```

Comment:
We should be able to remove ref to rsyslog as a collector all-to-gether.  We do not do any active development or support cc @sichvoge

Reply:
Removed. That doc was commented out cruft.